### PR TITLE
feat: benchmarking example

### PR DESCRIPTION
**Pull request by lino-levan**
This PR also added the capability to change the subcommand (which is obviously necessary in the case of `deno bench`).

### CODE REVIEW

File: data/benchmarking.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+// The most basic form of deno benchmarking is providing a name and an
+// anonymous function to run.
+Deno.bench("URL parsing", () => {
+  new URL("https://deno.land");
+});
+
+// We are also able to use an async function.
+Deno.bench("Async method", async () => {
+  await 1;
```
Comment (kt3k) [inline data/benchmarking.ts:22]: I think it's better to call actual async APIs here. How about `await crypto.subtle.digest('SHA-256', new Uint8Array([1, 2, 3]))`, for example, which doesn't require any permission

File: data/benchmarking.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
+/**
+ * @title Benchmarking
+ * @difficulty beginner
+ * @tags cli
+ * @run deno bench --allow-net <url>
```
Comment (kt3k) [inline data/benchmarking.ts:5]: Is `--allow-net` unnecessary?

Comment (lino-levan) [reply data/benchmarking.ts:5]: The async method that you also commented on used to be fetch before I decided not to do that, so this was necessary then. Nice catch!

File: data/benchmarking.ts [hunk 3]
```diff
@@ -0,0 +1,50 @@
+/**
+ * @title Benchmarking
+ * @difficulty beginner
```
Comment (kt3k) [inline data/benchmarking.ts:3]: @lino-levan How do you decide on the difficulty?

While `Deno.bench` is very easy to use, I wonder if a beginner would want to benchmark their code.

cc @lucacasonato

Comment (kt3k) [reply data/benchmarking.ts:3]: Let's revisit this later

**Review by kt3k — APPROVED** [review]

**Pull request merged**